### ISSUE TITLE
Refactored Schemas

### ISSUE THREAD

**Issue by cw-dvr8** [Sage-Bionetworks/sysbioDCCjsonschemas#78]
@Aryllen I'm refactoring the PsychENCODE schemas to use the current synapseAnnotations and putting them into a branch called refactored-schemas.  I'm dividing them into metadata_template and validation_schema and am naming the schemas accordingly.  My plan is to use the metadata_template schemas to generate the metadata templates, and then pulling those into the validation_schema schemas so that a) the metadata_template schemas can be consortium-independent, and b) I don't have to update the validation_schema schemas when I add new terms to the templates.

Please take a look at the assay metadata_templates.  If any of them match the ones you use in AMP, go ahead and pull them into a common folder as we discussed on Monday.

**Comment by Aryllen** [comment]
@cw-dvr8, I am checking them out right now. Based off a quick glance, I think there are some that are similar that we _could_ share, but are currently a little different. My thoughts would be combining to have the full key set for both templates. What do you think about that?

Also, for 'required' in these schemas, did you intend to put all of the properties in this or are you limiting the properties intentionally (example: ChIPseq has more properties than is required)?

**Comment by Aryllen** [comment]
Just finished skimming them and opening a PR for the few that seemed close enough for AD purposes. As mentioned above, I have some questions about how we should handle ones that are similar, but not quite close enough to automatically put them in the shared folder. Some of them we could potentially share are:
- ChIPseq
- RNAseq
- TMTquantitation
- individual (animal)
- manifest (if Mette wants AD to use 'isMultiIndividual')
- assay (generic -- if Mette wants AD to use, although I would probably not want to)
- biospecimen

The individual (human) template is too different from ours and uses slightly different terms. I think it would be a lot of work to change the AD one and it would cause conflict with existing metadata.

I have no idea what the WGBS one is. What is this for?

**Comment by cw-dvr8** [comment]
We'd have to look at what is different between the AD schemas and the PEC schemas, and then run them by Kelsey and Mette.  Kelsey is the lead for PEC so I don't want to make a call for her.

Also, I don't think that you're looking in the correct repo.  The WGBS assay has been changed to bisulfiteSeq and the generic assay template has been removed.

**Comment by Aryllen** [comment]
Ah, I thought the master branch had the current versions and that the refactored branch only had the new reference changes. I will have to go through the other branch, it seems.

I am hoping to get started on changing the AD templates to schemas this week. I'll track what's different in detail so we can run it by Mette and Kelsey.

**Comment by cw-dvr8** [comment]
I believe the refactored branch is the same as far as the fields go.  It just points to synapseAnnotations instead of sysbioDCCjsonschemas, so you shouldn't have to do a lot of rechecking.  I renamed a couple of the schemas to bring them up to the standard naming convention we've adopted.  Kelsey and Mette had me get rid of the generic assay schema also, they didn't want people to be able to upload assay data without us knowing about it.
